Automattic/wp-calypso
# Stats: Updating purchase page

Related to https://github.com/Automattic/jetpack/issues/31344.

## Proposed Changes

<img width="621" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4044428/95601685-cc76-47a9-8a9f-9914b15f7301">

* Updating CSS to match the final design
* Fixing typos and updating copy to the final version
* Code refactoring
* Localising currency symbols and display

## Testing Instructions



* Open the live branch for this PR
* Navigate to `/stats/purchase/:siteSlug?flags=stats/paid-stats`
* Change your account's language and see if the currency format changes. While you can't change the symbol, some languages replace the decimal sign e.g. `.` with `,`.

## Pre-merge Checklist
